config/locales/de.yml

```diff
@@ -434,9 +434,9 @@ de:
         address_html: '
```

Comment:
@KimberleyCook because the new address has a single quote `'` in 101 King's Cross Road, you now have to update the quotes that wrap the `address_html` entry to double quotes `"`.

Reply:
Oh gotcha, thanks. Should be done now